README.md

```diff
@@ -245,6 +252,30 @@ CREATE TABLE "ecommerce"."products" (
 
 This will produce the desired mock data to the topic `ecommerce.products`.
 
+#### Producing to Postgres
+
+You can also produce the data to a Postgres database. To do this, you need to specify the `-f sql` option and provide Postgres connection information in the `.env` file. Here is an example `.env` file:
+
+```
+# Postgres
+POSTGRES_HOST=
+POSTGRES_PORT=
+POSTGRES_DB=
+POSTGRES_USER=
+POSTGRES_PASSWORD=
+```
+
+Then, you can run the following command to produce the data to Postgres:
+
+```bash
+datagen \
+    -s tests/products.sql \
+    -f sql \
```

Comment:
again, we might consider using `-f postgres` to indicate that the data is landing in postgres